master/internal/api_config_policies.go

```diff
@@ -83,16 +83,52 @@ func (a *apiServer) GetWorkspaceConfigPolicies(
 		return nil, err
 	}
 
-	if !configpolicy.ValidWorkloadType(req.WorkloadType) {
-		errMessage := fmt.Sprintf("invalid workload type: %s.", req.WorkloadType)
-		if len(req.WorkloadType) == 0 {
+	resp, err := a.GetConfigPolicies(ctx, ptrs.Ptr(int(req.WorkspaceId)), req.WorkloadType)
```

Comment:
Can we pass `curUser` into `GetConfigPolicies` so that we don't have to redefine the variable there?
```suggestion
	resp, err := a.GetConfigPolicies(ctx, curUser, ptrs.Ptr(int(req.WorkspaceId)), req.WorkloadType)
```

Reply:
I don't think I am using the user variable inside getConfigPolicies?

Reply:
You're absolutely right, I was looking at `GetGlobalConfigPolicies` and `GetWorkspaceConfigPolicies` 🙈